src/States/MapViewStateDraw.cpp

```diff
@@ -251,17 +247,15 @@ void MapViewState::drawNavInfo()
 	if (isPointInRect(MOUSE_COORDS, MOVE_DOWN_ICON))
 	{
 		r.drawSubImage(mUiIcons, MOVE_DOWN_ICON.x(), MOVE_DOWN_ICON.y(), 64, 128, 32, 32, 255, 0, 0, 255);
-	}
-	else
+	} else
```

Comment:
There's quite a few style changes like this, which should really be done in their own commit.

----

Not too sure how Leeor feels about the style used here. I prefer a more compact style, though I'm uncertain what the code base tends to use.